src/app/core/ProductService.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { Product } from '../models/Product';
+import { Observable } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ProductService {
```

Comment:
tworzyles ten serwis ręcznie zamiast cli? :) bo nazwa pliku dziwna. zazwyczaj angular tworzy to w postaci product-service.ts (male litery, wyrazy poprzedzielane myslnikiem)

Reply:
Ekhm tak, zrobilem recznie. Wygeneruje przez CLI.